.appveyor.test.py

```diff
@@ -14,11 +14,11 @@
 
 
 def main():
-    unicode_stdout_stderr()
     logging.basicConfig(level=logging.INFO)
 
     sys.stdout = os.fdopen(sys.stdout.fileno(), 'w', 0)
     sys.stderr = os.fdopen(sys.stderr.fileno(), 'w', 0)
+    unicode_stdout_stderr()
```

Comment:
Inline what you currently have for `unicode_stdout_stderr` since you are going to break it for linux like this.